contrib/guix/guix-attest

```diff
@@ -153,10 +153,17 @@ for outdir in "${OUTDIRS[@]}"; do
         outdirs_already_attested_to+=("$outdir")
     else
         mkdir -p "$outsigdir"
-        echo "${outname}: Hashing build outputs to produce SHA256SUMS"
+
         (
             cd "$outdir"
-            files="$(find . -type f)"
+
+            if [ -e inputs.SHA256SUMS ]; then
+                echo "${outname}: Including existent input SHA256SUMS"
+                cat inputs.SHA256SUMS >> "$outsigdir"/SHA256SUMS
+            fi
```

Comment:
shouldn't this fail when the file is missing?

Reply:
Thinking about this since `dist-archive` does not have an `inputs.SHA256SUMS`... Right now, we attest to the bitcoin repo's git-archive in `distsrc-archive`, but I think perhaps it only makes sense to attest to it in the context of it being inputs to the host-specific outputs. When we add codesignatures, similarly it only makes sense to attest to the detached-sigs repo's git-archive in the context of it being inputs to the host-specific codesigned outputs. Will make a change!